pallets/xcmp-handler/src/lib.rs

```diff
@@ -197,13 +197,17 @@ pub mod pallet {
 		) -> DispatchResultWithPostInfo {
 			ensure_root(origin)?;
 
-			if currency_id != T::GetNativeCurrencyId::get() {
-				Err(Error::<T>::CurrencyChainComboNotSupported)?
+			match xcm_data.flow {
```

Comment:
There is more work required to support *any* currency for these fees. Right now the support is only for native currencies.